src/async_impl/request.rs

```diff
@@ -199,6 +227,14 @@ impl RequestBuilder {
         self
     }
 
+    /// Sets the request timeout.
```

Comment:
We should document that this timeout applies only to this request, and overrides any client timeout set.

Reply:
Updated documentation. I'm not a native speaker, so forgive me if something sounds weird.